### ISSUE TITLE
Memory leak in realtime container?

### ISSUE THREAD

**Issue by SharkyRawr** [appwrite/appwrite#1605]
## 🐛 Bug Report

Appwrite Realtime container memory usage is steadily increasing until the realtime container is restarted.

## Have you spent some time to check if this issue has been raised before?

Googled and looked at github issues, could not find anything related.

## To Reproduce

Run appwrite docker-compose.yml.  
Over time, memory usage steadily increases.

## Expected behavior

Steady memory usage, since I'm not actively using the installation or the realtime API.

## Actual Behavior

Memory usage increases over time:

![docker_memory-week](https://user-images.githubusercontent.com/246984/132583529-40634886-5dfe-400d-9cb1-db7d8210c088.png)

## Your Environment

```
Server: Docker Engine - Community
 Engine:
  Version:          20.10.8
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.16.6
  Git commit:       75249d8
  Built:            Fri Jul 30 19:52:33 2021
  OS/Arch:          linux/amd64
  Experimental:     true
 containerd:
  Version:          1.4.9
  GitCommit:        e25210fe30a0a703442421b0f60afac609f950a3
 runc:
  Version:          1.0.1
  GitCommit:        v1.0.1-0-g4144b63
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
```

Docker-compose.yml from https://github.com/appwrite/appwrite/blob/master/docker-compose.yml

**Comment by SharkyRawr** [comment]
![image](https://user-images.githubusercontent.com/246984/132583808-f936419a-f000-4b9c-9100-e57b35491242.png)

**Comment by TorstenDittmann** [comment]
Interesting 🤔

So you haven't used the Realtime service at all and it gradually increase in memory?

Thanks for the report, will investigate this asap :+1:

**Comment by TorstenDittmann** [comment]
This has been fixed in #1606 and released with [`0.10.3`](https://github.com/appwrite/appwrite/releases/tag/0.10.3).

Thanks a lot for the report 🙏🏻
